types/common.ts

```diff
@@ -3,12 +3,18 @@ export interface Location {
   path: string;
 }
 
-export type PageNames = 'main' | 'communication' | 'notice' | 'latestNews';
+export enum PageNames {
```

Comment:
사실, 이렇게 완전히 동일한 형태라면 굳이 enum을  사용할 필요가 있을지 의문이 들었습니다!

Reply:
원래 pagesUsingTag를
```
type PagesUsingTag = 'notice' | 'news`;
```
이렇게 했었는데 notice랑 news 하드코딩을 피하려고 enum으로 바꿔보기는 했습니다만 저도 어떤 쪽이 나은지 잘 판단이 안 서네요😢😢 이 정도 하드코딩은 감수하고 enum 안 쓰는 편이 나으려나요...?!

Reply:
책에서 아래 내용을 읽은 적이 있는데 union type으로 그대로 놔두는건 어떤가요!

> ...‘These divergent experiences for JavaScript and TypeScript users are a reason to avoid string-valued enums.
> 
> TypeScript offers an alternative to enums that is less common in other languages: a union of literal types.’
> 
> Effective TypeScript - Dan Vanderkam

Reply:
저도 `main="main`과 같이 동일한 형식이라면 그대로 놔두는것도 좋을것 같습니다.